core/domain/fs_domain.py

```diff
@@ -162,7 +162,9 @@ def get(self, filepath):
             return None
 
     def commit(self, filepath, raw_bytes, mimetype):
-        """Args:
+        """Commit to file.
```

Comment:
Write a clearer docstring that helps provide an explanation of what the method does. Don't just repeat the name of the method.